docs/user/datastores/analytic_queries.rst

```diff
@@ -263,14 +263,6 @@ following query hints:
 +-------------------------------------+--------------------+------------------------------------+
 | QueryHints.ARROW_DICTIONARY_FIELDS  | String (optional)  | dictionaryFields                   |
 +-------------------------------------+--------------------+------------------------------------+
-| QueryHints.ARROW_DICTIONARY_VALUES  | String (optional)  | N/A                                |
```

Comment:
Hmm....  Should we wait on removing the documentation?

Reply:
ah, not sure. i assume we don't want people to use it since it's going away. i guess we could optionally add some kind of deprecated flag to the docs?

Reply:
I doubt that anyone is using these features, so yeah, I'll merged this as is!